finesse/config.py

```diff
@@ -36,9 +36,15 @@
 EM27_URL = "http://10.10.0.1/diag_autom.htm"
 """The URL of the EM27 monitoring web server."""
 
-EM27_PROPERTY_POLL_INTERVAL = 60.0
+EM27_SENSORS_POLL_INTERVAL = 60.0
```

Comment:
Is this something we migth want configurable?

Reply:
Yep! Atm you can only have provide a `list` of possible values for device params, but for this case it would be better to have a text box/spin box. I've added an issue for this (#377) and once that's done it'll be easier to implement.